pages.de/linux/arch-chroot.md

```diff
@@ -3,18 +3,18 @@
 > Erweiterter `chroot`-Befehl zur Unterstützung des Arch Linux-Installationsprozesses.
 > Weitere Informationen: <https://man.archlinux.org/man/arch-chroot.8>.
 
-- Starte eine interaktive Shell (`bash` ist der Standard) in einem neuen Root-Verzeichnis:
+- Starte eine interaktive Shell (Standardmässig `bash`) in einem neuen Root-Verzeichnis:
```

Comment:
```suggestion
- Starte eine interaktive Shell (Standardmäßig `bash`) in einem neuen Root-Verzeichnis:
```
Standardmässig was used in the old spelling. The `ss` got replaced with an `ß`. ([source](https://www.duden.de/rechtschreibung/standardmaeszig))